rbc/tests/heavydb/test_text_encoding_dict.py

```diff
@@ -114,6 +115,28 @@ def fn_copy(mgr, t):
             str = mgr.getString(db_id, dict_id, t)
             return mgr.getOrAddTransient(mgr.TRANSIENT_DICT_DB_ID, mgr.TRANSIENT_DICT_ID, str)
 
+        @heavydb('TextEncodingNone(RowFunctionManager, TextEncodingDict)', devices=['cpu'])
+        def to_text_encoding_none_1(mgr, t):
+            db_id = mgr.getDictDbId('to_text_encoding_none_1', 0)
+            dict_id = mgr.getDictId('to_text_encoding_none_1', 0)
+            str = mgr.getString(db_id, dict_id, t)
+            n = len(str)
+            r = TextEncodingNone(n)
+            for i in range(n):
+                r[i] = str[i]
+            return r
```

Comment:
Why there is two UDFs with identical implementations?
Did you mean:
```suggestion
            return str
```
?

Reply:
Oops, my bad. Fixed